src/Chapter3.hs

```diff
@@ -752,6 +830,17 @@ parametrise data types in places where values can be of any general type.
 🕯 HINT: 'Maybe' that some standard types we mentioned above are useful for
   maybe-treasure ;)
 -}
+data Dragon a = Dragon { dragonMagicalPower :: a }
```

Comment:
Now, after you learned about `newtype`s, you can even make this data type as `newtype` :slightly_smiling_face: